README.md

```diff
@@ -56,7 +56,7 @@ For example, if you'd like to select the last successful build (stable or unstab
  
 ```groovy
 def runWrapper = selectRun job: 'upstream-project-name', 
- selector: [$class: 'StatusRunSelector', buildStatus: 'Successful'] 
+ selector: status('Successful') 
```

Comment:
Hmm... `Symbol` isn't applicable to enum values...
I think I should improve `Symbol`.